src/components/NavBar/MobileNavigation.js

```diff
@@ -0,0 +1,35 @@
+import NavLinks from "./NavLinks";
+import styles from "./NavBar.module.css";
+import { FiMenu } from "react-icons/fi";
+import { IoMdClose } from "react-icons/io";
+import { useState } from "react";
+
+const MobileNavigation = () => {
+  const [open, setOpen] = useState(false);
+  const hamburgerIcon = (
+    <FiMenu
+      className={styles.Hamburger}
+      size="20px"
+      color="white"
+      onClick={() => setOpen(!open)}
+    />
+  );
+
+  const closeIcon = (
+    <IoMdClose
+      className={styles.Hamburger}
+      size="20px"
+      color="white"
+      onClick={() => setOpen(!open)}
+    />
+  );
+
+  return (
+    <nav className={styles.MobileNavigation}>
+      {open ? closeIcon : hamburgerIcon}
+      {open && <NavLinks />}
```

Comment:
to nie bardzo działa, bo zwracasz na otwartym menu ten sam komponent, którego uzywasz na normalnej desktopowej belce, jesli tak, to powinien miec przekazany jakis parametr który mu powie, że powinien zachowywać się inaczej

Reply:
Dopytam, bo mam wątpliwości :) 
NavLinks jest ten sam dla Navigation i MobileNavigation, natomiast jest inaczej stylowany na poziomie ciała `@media`, gdzie mam np. stylowanie dla taga `ul { .... flex-direction: column;` i siup, jest kolumna.
A może jest inna praktyka?

Reply:
flex-direction: column jest ok, ale on nadal bedzie próbował się gnieździć wewnątrz navbara, dla tego Ci go tak rozciąga. Żeby wyłaził, potrzebujesz dać mu jakiś inny atrybut position:, ale ja i tak zamiast tego proponowalbym uzyc jakiegoś "button menu", cos jak tutaj: https://mui.com/material-ui/react-menu/#customization

Reply:
Użyłam material-ui i wygląda o wiele lepiej.